doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,71 @@ Tracing                                Jaeger
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.011 | The Monitoring service must support selective enabling and            |                                                       |
+|             | disabling of alerts by resource, category of resources, time periods. |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.012 | The Monitoring service should be implemented for high availability    |                                                       |
+|             | to ensure non-stop monitoring of critical infrastructure components   |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.013 | The Monitoring service should run as separately from production       |                                                       |
+|             | services                                                              |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.014 | Failure of the system being monitored should not cause a failure      |                                                       |
+|             | in the monitoring service                                             |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.015 | An inoperative monitoring service should not generate alerts about    |                                                       |
+|             | the monitored system                                                  |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.016 | The monitoring service must itself be monitored through a logging     |                                                       |
+|             | service                                                               |                                                       |
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| pas.mon.017 | The monitoring service must publish its APIs for programmatic         |                                                       |
```

Comment:
@pgoyal01 This requirement should appear higher on the table, before the optional requirements

Reply:
@karinesevilla Thanks -- moved this and another requirement higher in the table and moved a recommendation (should) lower.